exchanges/okx/okx_type_convert.go

```diff
@@ -142,30 +142,30 @@ func (a *Instrument) UnmarshalJSON(data []byte) error {
 		Alias: (*Alias)(a),
 	}
 
-	er := json.Unmarshal(data, chil)
-	if er != nil {
-		return er
+	err := json.Unmarshal(data, chil)
+	if err != nil {
+		return err
 	}
 	var val float64
-	if val, er = strconv.ParseFloat(chil.MaxLeverage, 64); er == nil {
+	if val, err = strconv.ParseFloat(chil.MaxLeverage, 64); err == nil {
```

Comment:
I used this because some of them might be empty strings or missing when queried from the server so I had to replace those empty strings with a zero value.